api/organisations/models.py

```diff
@@ -267,6 +267,9 @@ def save_as_free_subscription(self):
         if not getattr(self.organisation, "subscription_information_cache", None):
             return
 
+        if not self.organisation.subscription_information_cache.id:
+            return
```

Comment:
Should we add a comment here linking to the sentry issue? Maybe the git history is sufficient. I'll leave it up to your decision.

Reply:
Yeah a comment here certainly makes sense. Good call.